### ISSUE TITLE
HTML previewer doesn't render bold entries in tables

### ISSUE THREAD

**Issue by david-cortes** [rstudio/rstudio#3034]
When previewing HTML files generated from `knitr`, if these files contain some HTML table having entries in bold face and css style "united", then it doesn't show them as bold, while opening the same file in a modern web browser renders them in bold as expected.

**Steps to reproduce:**
* Generate some Rmarkdown document.
* Set the theme to "united", e.g. start it like this:
``` R
title: "html doc"
output:
  html_document:
    theme: united
    toc: yes
    toc_float:
      collapsed: no
```
* Generate some HTML table on it that would have entries in bold, e.g. a chunk like this:
``` R
library(knitr)
df = data.frame(col1 = c('a', 'b', 'c'), col2=c(1,2,3))
html_table = kable(df)
html_table
```

**Expected behavior:** entries in bold from the HTML should be shown in bold
**Actual behavior:** renders the file showing the bold entries in normal text
**Reproducible:** always
**System info:**
* R version: 3.5.0
* RStudio version: 1.1.453
* OS: Debian buster/sid

In the example above, the column names should be in bold. If putting `theme: yeti` they will be shown like that within RStudio. The HTML that it produces is as follows:
``` html
<table>
 <thead>
  <tr>
   <th style="text-align:left;"> col1 </th>
   <th style="text-align:right;"> col2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> a </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> b </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> c </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
</tbody>
</table>
```

The type `<b>` has the same issue in theme united, e.g. a chunk like this would also not show in bold:
``` R
library(knitr)
library(kableExtra)
df = data.frame(col1 = c('a', 'b', 'c'), col2=c(1,2,3))
html_table = kable(df)
html_table = gsub("<thead>.*</thead>", "", html_table)
html_table = gsub('(\\s+)<td style="text-align\\:left;"> (.*) </td>(\\s+)<td',
           '\\1<td style="text-align\\:left;"><b> \\2 </b></td>\\3<td',
           html_table, perl=TRUE) 
html_table = kable_styling(html_table)
html_table
```
``` html
<table class="table" style="margin-left: auto; margin-right: auto;">
 
<tbody>
  <tr>
   <td style="text-align:left;"><b> a </b></td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"><b> b </b></td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"><b> c </b></td>
   <td style="text-align:right;"> 3 </td>
  </tr>
</tbody>
</table>
```

**Comment by jmcphers** [comment]
I think this is an issue specific to RStudio 1.1, which uses an older rendering engine that has trouble rendering multiple weights of otherwise identical Web font families. Would you be willing to try our preview release of 1.2 to see if it resolves your issue?

https://www.rstudio.com/products/rstudio/download/preview/

**Comment by david-cortes** [comment]
Indeed, seems the issue is solved in the preview version 1.2.747.

**Comment by jmcphers** [comment]
Great, glad to hear it!
